akka-typed/src/main/scala/akka/typed/cluster/ClusterSingleton.scala

```diff
@@ -133,3 +135,91 @@ trait ClusterSingleton extends Extension {
   ): ActorRef[A]
 
 }
+
+object ClusterSingletonManagerSettings {
```

Comment:
need rebase? this should have been merged to master

Reply:
This was rebased, the ClusterSingletonManagerSettings was not ported to a typed version though: https://github.com/akka/akka/pull/23613/files